http/src/lib.nr

```diff
@@ -1,14 +1,11 @@
 use std::hash::poseidon::bn254::hash_1;
+pub mod data;
+pub mod tests;
 
-comptime global CARRIAGE_RETURN: u8 = "\r".as_bytes()[0];
-comptime global LINE_FEED: u8 = "\n".as_bytes()[0];
-comptime global COLON: u8 = ":".as_bytes()[0];
-comptime global SPACE: u8 = " ".as_bytes()[0];
-
-// comptime global CARRIAGE_RETURN: Field = 13;
-// comptime global LINE_FEED: Field = 10;
-// comptime global COLON: Field = 58;
-// comptime global SPACE: Field = 32;
+comptime global CARRIAGE_RETURN: u8 = "\r".as_bytes()[0]; // 13
+comptime global LINE_FEED: u8 = "\n".as_bytes()[0]; // 10
+comptime global COLON: u8 = ":".as_bytes()[0]; // 58
+comptime global SPACE: u8 = " ".as_bytes()[0]; // 32
 
 // todo: pack parser into field
```

Comment:
This could be done with an `.into()` method fairly simply. Maybe Noir has some utility packages that allow you to walk through fields of a struct like serde/strum would?

Reply:
I searched but there is no package handling enum in noir. (like strum in Rust) 😢